packages/bot-web-ui/src/components/trade-animation.jsx

```diff
@@ -15,19 +15,45 @@ const CircularWrapper = ({ className }) => (
     </div>
 );
 
-const AnimationInfo = () => (
-    <Popover
-        className='animation__info'
-        alignment='bottom'
-        message={localize(
-            'Stopping the bot will prevent further trades. Any ongoing trades will be completed by our system. Please be aware that some completed transactions may not be displayed in the transaction table if the bot is stopped while placing trades. You may refer to the statement page for details of all completed transactions.'
-        )}
-        zIndex={5}
-        should_disable_pointer_events
-    >
-        <Icon icon='IcInfoOutline' id='db-animation__clear-stat' />
-    </Popover>
-);
+const AnimationInfo = props => {
+    const { toggleAnimationInfoModal } = props;
+    return (
+        <div className='animation__info' onClick={toggleAnimationInfoModal}>
+            <Icon icon='IcInfoOutline' id='db-animation__clear-stat' />
+        </div>
+    );
+};
+
+const AnimationInfoModal = props => {
+    const { is_mobile, is_animation_info_modal_open, toggleAnimationInfoModal } = props;
```

Comment:
Isn't it better to do destructuring right in place of passed `props` instead of writing it in a separate line?

`const AnimationInfoModal = ({ is_mobile, is_animation_info_modal_open, toggleAnimationInfoModal }) => {`